docs/rpc.md

```diff
@@ -78,14 +78,14 @@ which would yield the response:
 
 ##### `invokefunction`
 
-neo-go's implementation of `invokefunction` does not return `tx`
+neo-go implementation of `invokefunction` does not return `tx`
 field in the answer because that requires signing the transaction with some
-key in the server which doesn't fit the model of our node-client interactions.
-Lacking this signature the transaction is almost useless, so there is no point
+key in the server, which doesn't fit the model of our node-client interactions.
+The transaction lacking this signature is almost useless, so there is no point
```

Comment:
Doesn't seem to be appropriate to me.

Reply:
Just doesn't sound right for this case (even though it's grammatically correct, of course). We're trying to explain causality here.

Reply:
`If this signature is lacking, the transaction is almost..` sounds more correct both grammatically and semantically to me.